lib/api.js

```diff
@@ -30,9 +33,9 @@ class API {
      *
      * @returns {undefined}
      */
-    constructor(config) {
+    constructor(name, config) {
         this.config = new Config(config);
-        this.Logger = Logger.bind(null, config);
+        this.Logger = new Logger(name, config);
```

Comment:
nope, this won't work: You need to provide a constructor here, not an instance; if you want the backwards compatibility with the former API, where the user can do:
```js
const werelogs = require('werelogs');
const logger = new werelogs.Logger('name'); // requires werelogs.Logger to be a constructor function
```

Additional comment here: Doing this means that one API can only provide one module logger; which was not the point of my change originally. I wanted to keep the possibility to have multiple module loggers; while having one `API` top layer that would federate the configuration for all the related module Loggers.